nitransforms/tests/test_io.py

```diff
@@ -37,6 +38,22 @@ def test_VolumeGeometry(tmpdir, get_testdata):
     assert len(vg.to_string().split("\n")) == 8
 
 
+def test_VG_from_LTA(data_path):
+    """Check the affine interpolation from volume geometries."""
+    # affine manually clipped after running mri_info on the image
+    oracle = np.loadtxt(StringIO("""\
+-3.0000   0.0000  -0.0000    91.3027
+-0.0000   2.0575  -2.9111   -25.5251
+ 0.0000   2.1833   2.7433  -105.0820
+ 0.0000   0.0000   0.0000     1.0000"""))
+
+    lta_text = "\n".join(
+        (data_path / "bold-to-t1w.lta").read_text().splitlines()[13:21]
+    )
+    r2r = VG.from_string(lta_text)
+    assert np.allclose(r2r.as_affine(), oracle, rtol=1e-4)
```

Comment:
Here's the kind of test I was thinking:

```suggestion
def test_volume_group_voxel_ordering():
    vg = VG.from_string("""\
valid = 1  # volume info valid
filename = no_file
volume = 5 6 7
voxelsize = 2 3 4
xras   = -1 0 0
yras   = 0 0 1
zras   = 0 -1 0
cras   = 0 0 0""")
    aff = vg.as_affine()
    assert np.allclose(vg["voxelsize"], [2, 3, 4])
    assert np.allclose(nb.affines.voxel_sizes(aff), [2, 3, 4])
    assert nb.aff2axcodes(aff) == ("L", "S", "P")
```

The property we're testing is that the zooms are correctly applied, so I'd rather be explicit than have it vague what qualities about the problematic geometry have been fixed.

Reply:
Sure, having an extra test and it being clear can only help.

However, I would keep the test I added for two reasons:
- First, because this is one example we know at some point of history failed.
- Second, because the test I wrote is dual to some test freesurfer folks could add to their repo (or, better said, I bet they already are doing something like that). Because `oracle` comes from executing `mri_info` (I saved the part of actually running it on every test), then we are able to at least ensure that our volume geometry interpretation is the same of freesurfer 6.0.1.

Reply:
Sure, we can keep your existing test.